src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -55,7 +60,7 @@ case class Model(
     val reps = replacements.map { r => ("replacements", r.toDoc) }
     val opts = options.map { o => ("options", o.toDoc) }
 
-    yamlMap(List(opts, deps, reps).collect { case Some(kv) => kv })
+    yamlMap(List(opts, deps, reps).collect { case Some(kv) => kv }, 2)
```

Comment:
Could probably do with a doc for the lines arg above, not sure what the `2` here does